packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -0,0 +1,121 @@
+@use "uswds-core" as *;
+@use "sass:color";
+
+.usa-in-page-nav-container {
+  align-items: flex-start;
+  display: flex;
+
+  // Set main landmark max width to ensure in-page nav will fit
+  main,
+  [class^="main"] {
+    max-width: units($theme-in-page-nav-main-content-max-width);
+    width: 100%;
+  }
+
+  .usa-in-page-nav__heading {
+    // TODO: This could be a setting?
+    @include typeset("ui", "2xs", 3);
+    @include u-font-weight("bold");
+    margin-bottom: units(2);
+    margin-top: 0;
+    padding: 0;
+  }
+
+  .usa-in-page-nav {
+    @include set-text-from-bg(
+      $bg-color: $theme-in-page-nav-background-color,
+      $context: "In-page navigation text color"
+    );
+    display: none;
+    margin-left: units($theme-in-page-nav-margin-left);
+    margin-top: units($theme-in-page-nav-margin-top);
+    max-width: units("card-lg");
+    order: 2;
+    position: sticky;
+    top: units($theme-in-page-nav-top);
+
+    @include at-media("tablet") {
+      display: block;
+      width: 100%;
+    }
+
+    nav {
+      @include set-text-and-bg(
+        $theme-in-page-nav-background-color,
+        $context: "In-page navigation background"
+      );    
```

Comment:
Empty spaces after semi-colon on 46.

![image](https://user-images.githubusercontent.com/3385219/200073292-4ea400e9-029c-4411-9205-91c3dcc4286d.png)

Reply:
@mejiaj also just pushed a fix for the extra spaces and formatting in the scss... Applied prettier to the file.